packages/devtools_app/lib/src/screens/vm_developer/vm_function_display.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import 'vm_developer_common_widgets.dart';
+import 'vm_object_model.dart';
+import 'vm_service_private_extensions.dart';
+
+// TODO(mtaylee): Finish implementation of [ICDataArrayWidget] and add it to
+// the [VmFuncDisplay].
+
+/// A widget for the object inspector historyViewport displaying information
+/// related to 'Func' (function) objects in the Dart VM.
```

Comment:
Nit: they're actually technically 'Function' objects, but we couldn't use that for `package:vm_service` since `Function` is a type already... :)